repose-aggregator/tests/functional-tests/src/integrationTest/configs/features/filters/simplerbac/multitenant/simple-rbac.cfg.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<simple-rbac xmlns="http://docs.openrepose.org/repose/simple-rbac/v1.0"
+             roles-header-name="X-Roles"
+             tenants-header-name="X-Tenant-Id"
```

Comment:
Does this need to be updated to case-match the roles below? I think I saw this being straight injected into the WADL, so I assume so.

Reply:
Good catch. Updating now.